FilePersistence/src/version_control/ListMergeComponent.cpp

```diff
@@ -172,17 +172,25 @@ LinkedChangesTransition ListMergeComponent::run(std::shared_ptr<GenericTree>& tr
 				for (auto elemId : chunk->spanBase_)
 				{
 					auto oldNode = treeBase->find(elemId);
-					auto newNode = treeA->find(elemId);
+					auto newNodeA = treeA->find(elemId);
+					auto newNodeB = treeB->find(elemId);
+					auto adjustChange = [&](GenericNode* newNode,
```

Comment:
Ok, if you want to leave the capturing in there that's fine, but since you really only intend to capture one variable I'd change this to the less error prone `[&oldNode]`